pipeline/pipeline_running/README.md

```diff
@@ -0,0 +1,94 @@
+# How to Run "The Pipeline"
+
+## Introduction
+
+In order to faciliate working with the pipeline a makefile is included in the code base under `pipeline/Makefile`. It is parametrized using a configuration file which by default is assumed to be in `pipeline/brca_pipeline_cfg.mk`.
+
+### Requirements
+
+In order to successfully run the pipeline using this setup your environment needs to be equipped with the following:
+
+* docker
+* GNU make >=v3.82
+* python with jinja2-cli installed (`pip install jinja2-cli`)
+
+TODO: comming soon, additional instructions from Mac OS X
+
+## Creating a New Data Release
+
+New data releases should ideally be generated on a dedicated pipeline machine. Although it could be run on any other machine in principle, some sources (e.g. LOVD) are only available from the pipeline machine.
+
+### Credentials
+
+Some early stages of the pipeline new some credentials to download data. These can be passed into the container by mounting an appropriate file. Also note, that some data sets are only available via the pipeline machine. However, later stages of the pipeline don't need any and some dummy file could be created.
+
+Currently, that a credential files should contain the following:
```

Comment:
Where and what name is this credentials file? docker-compose defaults to looking in .env and I usually include a template, sample_env that is checked into git and make sure .env is prevented from being commited in .gitignore. See https://github.com/UCSC-Treehouse/hub as a sample